### ISSUE TITLE
mpets can't export new pethead

### ISSUE THREAD

**Issue by BearNationSMP** [Pugabyte/BearNation#212]
Submitted by **Filid**

**Comment by FilidDamhsoir** [comment]
https://gyazo.com/321ac368859b56684f5591810a0b5ee0
Checked working pets to see if it was somethign I did wrong with the config, nothing that wakka and I can find.  
Checked the dev's discord.  The devs are useless. The last person with this issue got a reasouning "i dunno" from the Dev.
 *possibly an issue with the fact that we are nowhere near the newest version, but as we know this plugin has caused issues in the past, wakka and I didn't want to update it before talking to the pug- make sure there wasn't a specific reason why we are still on 1.5.whatever (ie- cause newer versions crashed more important stuff) rather than 2.1.1
